pages/orders/[id].vue

```diff
@@ -0,0 +1,63 @@
+<script setup>
+import { ref, watchEffect } from 'vue';
+import { useRoute } from 'vue-router';
+import ProductList from '../components/ProductList/ProductList.vue';
```

Comment:
After rename this file You can delete this line `import ProductList from '../components/ProductList/ProductList.vue';`